app/controllers/games_controller.rb

```diff
@@ -0,0 +1,38 @@
+class GamesController < ApplicationController
+  before_action :authenticate_user!, only: [:new, :create, :edit, :update, :destroy]
+
+  def new
+    @game = Game.new
+  end
+
+  def create
+    @game = Game.create!(game_params.merge(white_player_id: current_user.id))
```

Comment:
What sort of errors are you getting when you use `create`? If it's called properly and with valid data, it should not generate any exceptions, whereas `create!` will definitely produce an exception when validations fail. Since it's normal for validations to fail in production, this is a big distinction, and you shouldn't use `create!` unless the method is set up to rescue the resulting exception.

In other words, just go with `create`. :)